swagger.yaml

```diff
@@ -61,6 +61,37 @@ paths:
           description: Internal Server Error
           schema:
             $ref: '#/definitions/Error'
+  /locations:
+    get:
+      summary: List locations
+      description: Return a list of HR locations that can be related to a job record, including minimum wages if applicable.
+      parameters:
+        - in: query
+          name: date
+          description: "Date to search by for minimum wage data. Minimum wage depends on the date. Example: '2018-12-31'"
+          required: false
+          type: string
+          format: date
+          default: The current date
+        - in: query
+          name: state
+          description: "Filter by state code. Use ANSI standard INCITS 38:2009 for state codes."
+          required: false
+          type: string
+        - $ref: '#/parameters/pretty'
+      responses:
```

Comment:
As we discussed in person, I think it will be helpful if we can provide the "city" filter.

Reply:
@tsoliangwu0130 I thought about this and realized it may not be a good idea to add this. In the HRIS, the city column is being used for the name of the city if the state is oregon. If the location is non-oregon, the city column is being used for the name of the state 👎 

We could add some logic to overcome this, but for now, I don't think it's worthwhile to add a city parameter.